tests/enchancement_jisc_skeleton.php

```diff
@@ -32,6 +32,16 @@ abstract class enchancement_jisc_skeleton extends advanced_testcase {
      */
     protected $multipletestnumber = 5;
 
+    /**
+     * @var int Generated history-log events numbers
+     */
+    protected $generatedhistorylog = 11;
+
+    /**
+     * @var int Generated xapi-log events numbers
+     */
+    protected $generatedxapylog = 1;
+
```

Comment:
@lzabo Hi, should this be named $generatedxapilog instead of $generatedxapylog? Is it named $generatedxapylog to prevent a name conflict with another variable?

Reply:
👍